lang/en.json

```diff
@@ -743,6 +743,8 @@
         },
         "add_to_cart": "Add to Cart",
         "adding_to_cart": "Adding to cart…",
+        "add_to_wishlist_success_message": "The product has been successfully added to the wishlist. Show <a href=\"{wishlistPath}\">more details</a>.",
+        "add_to_wishlist_error_message": "Something went wrong.",
```

Comment:
could we use common.generic_error here?

Reply:
@BC-tymurbiedukhin yes, of course, I think this is a better place to refer to the error.

Reply:
@BC-tymurbiedukhin done!